packages/rum/test/benchmarks/server.js

```diff
@@ -93,4 +95,10 @@ module.exports = function startServer() {
       resolve(server)
     })
   })
-}
+})
+
+!(async () => {
+  if (require.main === module) {
```

Comment:
added this so that we can just start the server for testing with `node packages/rum/test/benchmarks/server.js`